core/src/main/java/org/elasticsearch/index/query/MoreLikeThisQueryParser.java

```diff
@@ -78,29 +55,44 @@
         ParseField FAIL_ON_UNSUPPORTED_FIELD = new ParseField("fail_on_unsupported_field");
     }
 
+    public MoreLikeThisQueryParser() {
+
+    }
```

Comment:
Sure I can remove. Note all parser seem to be having this empty unused constructor.